### PR TITLE
Fix animated logo

### PR DESCRIPTION
**Pull request by nathsimpson**
- Fix styles for animated logo, including supporting light/dark mode.
- Add Typescript types for P5

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,231 +1,229 @@
...
 export default function Home() {
-  return (
-    <main className="flex w-full min-h-screen flex-col items-center justify-between">
-      <SiteHeader />
-      <Container>
-        <div className="flex flex-col gap-3">
-          <AnimatedLogo />
+	return (
+		<main className="flex w-full min-h-screen flex-col items-center justify-between">
+			<Container>
```
Comment (nathsimpson) [inline src/pages/index.tsx:10]: All I did here was remove the SiteHeader, as we are sticking with one page for now.

File: src/components/animated-logo.tsx [hunk 2]
```diff
@@ -46,8 +45,27 @@ const sketch: Sketch = (p5: any) => {
...
...c-2.8-2.6-5-5.8-6.5-9.5s-2.3-7.8-2.3-12.3c0-4.7.7-8.9 2.2-12.7 1.5-3.8 3.6-7 6.3-9.6 2.7-2.6 6-4.7 9.8-6.1s8.2-2.1 13-2.1 9.1.7 12.9 2.1c3.8 1.4 7 3.5 9.7 6.2 2.6 2.7 4.7 6 6 10 1.4 3.9 2.1 8.3 2.1 13.3v1.9zm25.7 24.9h-19.9V45.7h19.9V137zm28.1 0h-19.9V80.6h19.9V137zm.3-73.8c-.6 1.4-1.4 2.5-2.4 3.6-1 1-2.2 1.8-3.6 2.4-1.4.6-2.8.9-4.3.9s-3-.3-4.3-.9c-1.4-.6-2.5-1.4-3.6-2.4-1-1-1.8-2.2-2.4-3.6-.6-1.4-.9-2.8-.9-4.3s.3-3 .9-4.3c.6-1.4 1.4-2.5 2.4-3.6 1-1 2.2-1.8 3.6-2.4 1.4-.6 2.8-.9 4.3-.9s3 .3 4.3.9c1.4.6 2.5 1.4 3.6 2.4 1 1 1.8 2.2 2.4 3.6.6 1.4.9 2.8.9 4.3s-.3 3-.9 4.3zm67.9 53c0 7.6-2.5 13.3-7.5 17.2-5 3.8-12.5 5.7-22.5 5.7s-17.5-1.9-22.5-5.7-7.5-9.5-7.5-17.2V80.6h19.9v31.8c0 7 3.4 10.5 10.1 10.5 6.7 0 10.1-3.5 10.1-10.5V80.6h19.9v35.6zM361.4 137h-19.9v-27.8c0-2.9-.2-5.2-.6-7s-1-3.3-1.8-4.3-1.7-1.7-2.7-2.1-2.1-.5-3.3-.5c-6.4 0-9.6 4.7-9.6 14v27.8h-19.9v-27.8c0-2.9-.2-5.2-.5-7.1-.4-1.9-.9-3.4-1.6-4.5s-1.6-1.9-2.7-2.3-2.4-.6-3.8-.6c-1.2 0-2.4.2-3.5.6s-2.1 1.1-2.9 2.1c-.8 1.1-1.5 2.5-2 4.4s-.7 4.3-.7 7.3V137H266V80.6h19.9v6.9c5.4-5.2 11.5-7.8 18.3-7.8 8.4 0 14.7 3.5 19.1 10.6 4.5-7.1 10.9-10.7 19.1-10.7 2.6 0 5.1.4 7.5 1.2 2.3.8 4.4 2 6 3.6 1.7 1.6 3 3.8 4 6.4s1.5 5.8 1.5 9.6V137z" />
```
Comment (nathsimpson) [inline src/components/animated-logo.tsx:66]: I use 'fill=currentcolor' to change the svg color in dark/light mode. 

I had to inline the svg code from selium-overlay.svg, as currentcolor doesn't work when using img.

File: src/pages/index.tsx (hunk 1 above)
Comment (itpetey) [reply src/pages/index.tsx:10]: Hehe, I'm actually in the process of putting in a news page so will need that header back 😅

File: src/components/animated-logo.tsx (hunk 2 above)
Comment (itpetey) [reply src/components/animated-logo.tsx:66]: Should selium-overlay.svg be deleted then?

**Review by itpetey — CHANGES_REQUESTED** [review]
Thanks so much for this! Just the two comments to address but otherwise it looks awesome!

**Review by itpetey — APPROVED** [review]

**Pull request merged**